Custom Op example using HIP kernel (#1200)
ROCmSoftwarePlatform/AMDMIGraphX: examples/migraphx/custom_op_hip_kernel/README.md

# Custom Kernel using MIGraphX API. 

>  This is a simple example that shows how to write custom implementation for any operator using MIGraphX's C/C++ APIs. 

Reviewer: ```suggestion
 This is a simple example of a custom operator implementation using MIGraphX's C/C++ APIs. 
```

>  Kernels can be written in either HIP, MIOpen, or by using RocBLAS library. 

Reviewer: ```suggestion
 Kernels can be written in either HIP, MIOpen, or by using RocBLAS library. This particular example uses **HIP**.
```
